llvm/test/TableGen/GlobalISelEmitterSkippedPatterns.td

```diff
@@ -23,6 +25,10 @@ def INSN : I<(outs GPR32:$dst), (ins GPR32:$src1, complex:$src2), []>;
 
 //===- Bail out when we define a variable twice wrt complex suboperands. -===//
 
+#ifdef IGNORE
+let GISelShouldIgnore = 1 in
+#endif
+// CHECK0-NOT: warning: Skipped pattern: Error: {{.*}}
```

Comment:
nit: Can you rename this to something like `IGNORED` ? `CHECK0` doesn't tell much at first glance